### ISSUE TITLE
Front end styling changes to adv. data page/feed template

### ISSUE THREAD

**Issue by PaulClark2** [18F/openFEC-web-app#2330]
Front end styling changes to adv. data page/feed template
+ add accordion
+ change feed itemstyles

**Comment by jenniferthibault** [comment]
🎨 In https://github.com/18F/openFEC-web-app/issues/2265 we made a [case study prototype of the Candidates page](https://gsa.invisionapp.com/share/QCDMN3I3D#/254874863_Advanced-Data_Candidates_Copy_2) to demonstrate the approach to integrating this data.

This prototype shows how we might distribute the different data sets (from the data catalog and the FTP). The accordions are clickable and indicate how we might label and group data by year, as well as how we can create the space to describe the data within it. For describing the bulk data, I've dropped in a placeholder paragraph to mark out the space we could use. (this task is tracked separately in https://github.com/18f/openfec-web-app/issues/2321 )

Design changes to the previous page pattern:
- Creates two sections in each tab: (1) Search or browse data; (2) Download data in bulk
- Removes "feed" styling (removes lines between each item and nested/structured/indented relationships between data sets
- Removes document icon for collected data sets
- Adds magnifying icon in front of each searchable data set: this helps keep the focus on these features
- Adds accordion that expands to hold downloadable data catalog files
- FTP data goes into the bulk data section in accordions
- Removes "Bulk data and other sources" as a left-hand menu category
